Makefile

```diff
@@ -18,7 +18,7 @@ TAR_EXT := tar.bz2
 TAR_FILENAME := $(NAME)-$(VERSION).$(TAR_EXT)
 TAR_LOC := .
 TAR_FILE := $(TAR_LOC)/$(TAR_FILENAME)
-GO_MIN_VERSION := 1.5.1
+GO_MIN_VERSION := 1.7
```

Comment:
I've added "Closes #875" to the main comment to close that issue when this gets merged.